package.json

```
@@ -0,0 +1,84 @@
+{
+  "name": "fashion24-back-end",
+  "version": "0.0.1",
+  "description": "",
+  "author": "team fashion-24 <https://github.com/fashion24-developer>",
+  "private": true,
+  "license": "UNLICENSED",
+  "contributors": [
+    "Biho Jeong <jjb26433@naver.com>"
+  ],
```

```suggestion
  "contributors": [
    "Biho Jeong <jjb26433@naver.com>",
    "Junhyuk Park <pjh_2004@naver.com>"
  ],
```

"Junhyuk Park <pjh_2004@naver.com>" 추가해주세용